src/acme/ui/AcmeUI.java

```diff
@@ -88,136 +100,138 @@ private void buildMenu() {
+        this.revalidate();
+        this.repaint();
+    }
+
     public void logoutUser() {
         company.setCurrentUser(null);
         this.loginScreen();
     }
 
-    // Everyone will tie in their panel like this.  Replace my example with your code
-    public void loginScreen() {
-        ExampleJPanel exampleJPanel = new ExampleJPanel();
-        this.setPanel(exampleJPanel);
-    }
-
-    // Everyone will tie in their panel like this.  Replace my example with your code
-    public void ticketList() {
-        ExampleJPanel exampleJPanel = new ExampleJPanel();
-        this.setPanel(exampleJPanel);
-    }
-
-    // Everyone will tie in their panel like this.  Replace my example with your code
-    public void ticketCreate() {
-        // This will be called with null to create a ticket
-        ExampleJPanel exampleJPanel = new ExampleJPanel();
-        this.setPanel(exampleJPanel);
-    }
-
-    // Everyone will tie in their panel like this.  Replace my example with your code
-    public void ticketComplete(Ticket ticket) {
+    // Everyone will tie in their panel like this.  Replace my example with your code
+    public void loginScreen() {
+        ExampleJPanel exampleJPanel = new ExampleJPanel();
+        this.setPanel(exampleJPanel);
+    }
+
+    // Everyone will tie in their panel like this.  Replace my example with your code
+    public void ticketList() {
+        ExampleJPanel exampleJPanel = new ExampleJPanel();
+        this.setPanel(exampleJPanel);
+    }
+
+    // Everyone will tie in their panel like this.  Replace my example with your code
+    public void ticketCreate() {
+        // This will be called with null to create a ticket
+        ExampleJPanel exampleJPanel = new ExampleJPanel();
+        this.setPanel(exampleJPanel);
+    }
+
+    // Everyone will tie in their panel like this.  Replace my example with your code
+    public void ticketComplete(Ticket ticket) {
         CompleteATicketUI ticketCompleteUI = new CompleteATicketUI(ticket);
         this.setPanel(ticketCompleteUI);
-    }
-
-    // Everyone will tie in their panel like this.  Replace my example with your code
-    public void customerList() {
+    }
+
+    // Everyone will tie in their panel like this.  Replace my example with your code
+    public void customerList() {
         CustomerListUI customerListUI = new CustomerListUI();
         this.setPanel(customerListUI);
-    }
-
-    // Everyone will tie in their panel like this.  Replace my example with your code
-    public void customerAddUpdate(Customer customer) {
-        // This will be called with null to create
-        ExampleJPanel exampleJPanel = new ExampleJPanel();
-        this.setPanel(exampleJPanel);
-    }
-
-    // Everyone will tie in their panel like this.  Replace my example with your code
-    public void userList() {
+    }
+
+    // Everyone will tie in their panel like this.  Replace my example with your code
+    public void customerAddUpdate(Customer customer) {

```

Comment:
This needs to create your add/update page

Reply:
Done.